packages/alore-auth-ui/src/auth/Register.tsx

```
@@ -0,0 +1,1175 @@
+        });
+      }
+    }
+  }
+
+  const isUserInfoSubmitDisabled = useMemo(() => {
+    const values = userInfoGetValues();
+
+    if (!values) return true;
+
+    if (!requireUsername) {
+      delete values.nickname;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)